From 06081bd53399b6c71c373a13bffc6ea990e833b0 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 25 Jun 2019 10:52:52 +0800
Subject: [PATCH] customize default option for OE

- Do not verify ssl certification by default

- Disable selinux by default

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Fix patch-fuzz error for anaconda 42 on 20250414.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/argument_parsing.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/argument_parsing.py b/pyanaconda/argument_parsing.py
index e24f0f1b99..e825a9b14c 100644
--- a/pyanaconda/argument_parsing.py
+++ b/pyanaconda/argument_parsing.py
@@ -490,7 +490,7 @@ def getArgumentParser(version_string, boot_cmdline=None):
 
     ap.add_argument("--addrepo", dest="addRepo", default=[], metavar="NAME,ADDITIONAL_REPO_URL",
                     action=ParseAddRepo, help=help_parser.help_text("addrepo"))
-    ap.add_argument("--noverifyssl", action="store_true", default=False,
+    ap.add_argument("--noverifyssl", action="store_true", default=True,
                     help=help_parser.help_text("noverifyssl"))
     ap.add_argument("--liveinst", action="store_true", default=False,
                     help=help_parser.help_text("liveinst"))
@@ -521,7 +521,7 @@ def getArgumentParser(version_string, boot_cmdline=None):
 
     from pyanaconda.core.constants import SELINUX_DEFAULT
     ap.add_argument("--noselinux", dest="selinux", action="store_const",
-                    const=SELINUX_DISABLED, default=SELINUX_DEFAULT,
+                    const=SELINUX_DISABLED, default=SELINUX_DISABLED,
                     help=help_parser.help_text("noselinux"))
 
     # Use a custom action to convert --selinux=0 and --selinux=1 into the
-- 
2.7.4

